### PR TITLE
Add docs page for `expo-mesh-gradient`

### PR DESCRIPTION
**Pull request by tsapeta**
# Why

Following up #33187

# How

- Added docs page
- Generated docs data
- Added a logo of the package
- Exported missing props type
- Added missing `@default` for `points` prop

# Test Plan

/versions/unversioned/sdk/mesh-gradient/

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/mesh-gradient.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: MeshGradient
+description: A module that exposes MeshGradient view from SwiftUI to React Native.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-video'
```
Comment (reichhartd) [inline docs/pages/versions/unversioned/sdk/mesh-gradient.mdx:4]:
```suggestion
sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-mesh-gradient'
```

File: docs/pages/versions/unversioned/sdk/mesh-gradient.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+packageName: 'expo-mesh-gradient'
+iconUrl: '/static/images/packages/expo-mesh-gradient.png'
+platforms: ['ios', 'tvos']
```
Comment (reichhartd) [inline docs/pages/versions/unversioned/sdk/mesh-gradient.mdx:7]:
```suggestion
platforms: ['ios', 'tvos', 'macos']
```

Comment (tsapeta) [reply docs/pages/versions/unversioned/sdk/mesh-gradient.mdx:7]: Hah, I know that macOS is listed as supported in the podspec, but I'd like to actually test it there before listing it in the docs 😅

**Review by Simek — APPROVED** [review]
Tested locally, LGMT! 👍

**Pull request merged**